pages/api/secrets/[id]/details.ts

```diff
@@ -24,19 +24,12 @@ export default async (
 
   try {
     const token = await bt.tokens.retrieve(id);
-    const datePlusTTL = add(parseISO(token.createdAt), {
-      seconds: Number.parseInt(token.metadata.ttl),
-    });
+    const expiresAt = parseISO(token.expiresAt);
 
-    if (getTime(datePlusTTL) > Date.now()) {
-      res.status(200).json({
-        id: token.id,
-        timeLeft: datePlusTTL,
-      });
-    } else {
-      await bt.tokens.delete(id);
-      res.status(404).json({});
-    }
+    res.status(200).json({
+      id: token.id,
+      timeLeft: expiresAt,
```

Comment:
are we actually using this? it doesn't look like we are. if we are, we would need to calculate the actual time left here

Reply:
we weren't idk why it was there, removed it